docs/topics/api-references.topic

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       id="api-references" title="API references">
+
+    <p>Welcome to the Kotlin API References page. Here you'll find links to the API documentation for the official Kotlin libraries and tools.</p>
+
+    <list columns="2">
+        <li>
+            <a href="https://kotlinlang.org/api/latest/jvm/stdlib/"><b>Standard Library (stdlib)</b></a>
+            <br/>
+            <p>The Kotlin Standard Library provides core functionality for Kotlin programming. It includes essential APIs such as collections, text and string processing, I/O, ranges, sequences, and more, and is available on all supported platforms.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/JetBrains/kotlin">kotlin</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/latest/kotlin.test/"><b>Test Library (kotlin.test)</b></a>
+            <br/>
+            <p>A multiplatform testing library that provides common testing annotations and utility functions. It supports integration with popular test frameworks on each platform and offers a unified testing experience across the Kotlin ecosystem.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/JetBrains/kotlin">kotlin</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/"><b>Coroutines (kotlinx.coroutines)</b></a>
+            <br/>
+            <p>A powerful library for asynchronous programming using Kotlin coroutines. It offers high-level coroutine builders, concurrency primitives, structured concurrency, and integration with reactive libraries and frameworks.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx.coroutines">kotlinx.coroutines</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-core/"><b>Serialization (kotlinx.serialization)</b></a>
+            <br/>
+            <p>A multiplatform JSON and binary serialization library. It provides a type-safe, compile-time mechanism for converting Kotlin objects to and from a variety of formats such as JSON, CBOR, and Protocol Buffers.</p>
+            <img src="github.svg" width="18" alt="GitHub"/> <a href="https://github.com/Kotlin/kotlinx.serialization">kotlinx.serialization</a>
+            <br/>
+        </li>
+        <li>
+            <a href="https://kotlinlang.org/api/kotlinx-io/"><b>Kotlin I/O Library (kotlinx-io)</b></a>
```

Comment:
For some libraries (like serialization or coroutines), these links leads to `*-core` modules, for others (like here) - to a top level doc. I think we should be consistent here and either always point to a core module (if one exists), or to a documentation root (I'd prefer the latter).

Reply:
Thank you! changed to the top-level doc